packages/blade/src/components/Typography/Code/Code.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Text } from '../Text';
+import { Code as CodeComponent } from './Code';
+
+const CodeStoryMeta: Meta = {
+  title: 'Components/Typography/Code',
+  component: CodeComponent,
+  args: {
+    size: 'medium',
+    type: 'subtle',
+    children: 'SENTRY_AUTH_TOKEN',
+  },
+};
+
+const CodeTemplate: ComponentStory<typeof CodeComponent> = (args) => (
+  // For React Native, use flex to align items correctly
+  <Text>
+    Lorem ipsum normal text <CodeComponent {...args} /> component
+  </Text>
+);
+
+export default CodeStoryMeta;
```

Comment:
Maybe we should add a kitchen sink example with all the variants.

Reply:
Was thinking that but we don't really have kitchen sink for other typography components and also the variants can be changed from docs controller itself and they don't visibility look that different because it's just change in size.

Reply:
Yes in fact we should add other kitchen sink too. 

My main reason of adding kitchen sink examples is to catch visual regressions with chromatic. 
If the storybook doesn't render the size and other variants chromatic won't screenshot them.

Reply:
Added the sink 👯